geode-assembly/src/test/java/org/apache/geode/test/junit/rules/HttpResponseAssert.java

```diff
@@ -87,23 +86,17 @@ public HttpResponseAssert statusIsOk() {
     return this;
   }
 
-  public JSONObject getJsonObject() {
-    JSONTokener tokener = new JSONTokener(responseBody);
-    return new JSONObject(tokener);
+  public JsonNode getJsonObject() throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    return mapper.readTree(responseBody);
   }
 
-  public JSONArray getJsonArray() {
-    JSONTokener tokener = new JSONTokener(responseBody);
-    JSONArray array = new JSONArray(tokener);
-    return array;
-  }
-
-  public ListAssert<Object> hasJsonArray() {
-    JSONTokener tokener = new JSONTokener(responseBody);
-    JSONArray array = new JSONArray(tokener);
+  public ListAssert<Object> hasJsonArrayOfDoubles() throws IOException {
```

Comment:
If this is named as hasJsonArrayOfDoubles should it return ListAssert<Double>?

Reply:
Good point. Did you mean it should return `ListAssert<Double>`?

Reply:
Oh, yeah, of course.